packages/bot/src/stores/run-panel-store.js

```diff
@@ -324,34 +323,18 @@ export default class RunPanelStore {
                 this.disposeIsSocketOpenedListener = reaction(
                     () => client.loginid,
                     (loginid) => {
+                        terminateAndClear();
                         if (loginid) {
-                            this.root_store.journal.pushMessage(localize('You have switched accounts.'));
+                            journal.pushMessage(switch_account_notification.message, message_types.ERROR);
```

Comment:
do we need to show the message in journal when there's already notification?

Reply:
@sam-binary yea we dont need to, i removed it from journal